lib/src/widgets/utils/ds_card.widget.dart

```diff
@@ -34,10 +34,12 @@ class DSCard extends StatelessWidget {
   Widget _resolveWidget() {
     switch (type) {
       case DSMessageContentType.textPlain:
+      case DSMessageContentType.sensitive:
         return DSTextMessageBubble(
-          text: content,
+          text: content is String ? content : '**********',
```

Comment:
Yes, because on this case, we have only 2 option `DSMessageContentType.textPlain` and `DSMessageContentType.sensitive`. If is not string, then is sensitive (and we must not show the content)